assets/libraries/serverlessfw.rego

```diff
@@ -0,0 +1,37 @@
+package generic.serverlessfw
+
+import data.generic.common as common_lib
+
+resourceTypeMapping(resourceType, provider)= resourceTypeVal{
+    resourceTypeVal :=resourcesMap[provider][resourceType]
+}
+
+resourcesMap = {
```

Comment:
Please, update in `https://github.com/Checkmarx/kics/blob/master/docs/utilities.md#resource-type-and-resource-name`